skimage/data/__init__.py

```diff
@@ -125,7 +125,12 @@ def create_image_fetcher():
     # This helps pooch understand that it should look in master
     # to find the required files
     pooch_version = __version__.replace('.dev', '+')
-    url = "https://github.com/scikit-image/scikit-image/raw/{version}/skimage/"
+    if '+' in pooch_version:
```

Comment:
why is thel ogic different?

Reply:
@hmaarrfk for master, the data lives in `github.com/scikit-image/scikit-image/raw/master/skimage/...`, while for version tags, it's in `github.com/scikit-image/scikit-image/raw/v0.18.0/skimage/...`, for example.

Technically, the approach is not quite correct: the v0.18.x branch will still look in master. This happens to work, but might not always. For now, it gets us unstuck. In the future, maybe we move to setuptools_scm, which would solve a lot of issues more elegantly.

Reply:
This, and pooch knows how to convert internally `version` to `master` so that in the url it's `master`, I guess?